package.json

```diff
@@ -90,5 +90,8 @@
     "url-parse": "^1.1.7",
     "velocity-animate": "^1.2.3",
     "yargs": "^6.6.0"
+  },
+  "optionalDependencies": {
```

Comment:
I don't think this should be an optional dependancy. For users of Linux, the code will bomb if this dependency is not present.

![image](https://user-images.githubusercontent.com/794517/39530233-279fcd54-4dee-11e8-945a-2902f965ba03.png)

If you really wanted it to stay as optional, then the code that imports it would have to handle any failures of it not being there. From the [npm docs](https://docs.npmjs.com/files/package.json#optionaldependencies):

```
It is still your program's responsibility to handle the lack of the dependency. For example, something like this:
```

Could you please either make it a required dependency in package.json -or- handle the failure of it not being there?

Reply:
Agreed, I just copied this from some other project where this dependency was used. I will make it a normal mandatory dependency.